package.json

```
@@ -1,12 +1,17 @@
 {
-  "name": "EngageSphere-Test-Design-Masterclass-turma-2",
+  "name": "engagesphere-test-design-masterclass",
   "version": "1.0.0",
   "description": "Sample project with a Node.js backend and a React frontend.",
   "scripts": {
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "api:run": "cypress run --spec cypress/e2e/API/**.cy.js",
+    "gui:run": "cypress run --spec cypress/e2e/GUI/**.cy.js",
```

```suggestion
    "gui:run": "cypress run --spec 'cypress/e2e/GUI/**.cy.js'",
```

O mesmo que o comentário anterior.